app/logic/volunteers.py

```diff
@@ -92,3 +98,26 @@ def setUserBackgroundCheck(user,bgType, checkPassed, datePassed):
     update = BackgroundCheck.create(user=user, type=bgType, passBackgroundCheck=checkPassed, datePassed=datePassed)
     update.save()
     createLog(f"Updated {user.firstName} {user.lastName}'s background check for {bgType} to {bool(checkPassed)}.")
+
+def isProgramManagerForEvent(user, event= None):
+    """
+    This function checks to see if a user is a program manager for a program.
+    NOTE: this function needs the event parameter to work
+    user: expects a user peewee object
+    event: expects an event peewee object
+    Returns: bool whether the appropriate information is given.
+    """
+
+    #neither event nor programId are passed
+    if not (event):
+        raise ValueError("Not enough parameters given to this function.")
+
+    #if event is passed but no programId is passed
+    if event:
+        programIdQuery = (ProgramEvent.get(ProgramEvent.event == event)).program
```

Comment:
The Event class has a property `singleProgram`, which you can use to get the program id from an event or None if there are multiple programs or no programs